rust/toolchain.bzl

```diff
@@ -49,17 +50,19 @@ def build_rustc_command(ctx, toolchain, crate_name, crate_type, src, output_dir,
           "--codegen ar=%s" % ar,
           "--codegen linker=%s" % cc,
           "--codegen link-args='%s'" % ' '.join(cpp_fragment.link_options),
+          "--codegen link-args='%s'" % ' '.join(cpp_fragment.mostly_static_link_options([], False)),
```

Comment:
Yeah, this is pretty unfortunate, but I can't find a way to get "whatever the right set of default C++ linking options are" from Bazel. The `link_opts` here are usually empty. The `fully_static...`, `mostly_static...`, and `dynamic_...` variants contain what I need, but are apparently deprecated, and also don't give you any insight into which of the three is appropriate in this situation.

What I really need is just whatever the appropriate C++ linker options are, based on the sysroot, embedded runtimes, etc. for the given C++ toolchain. For example in a default crosstool this will add `-lstdc++` during a `cc_binary` link, but the Rust link step doesn't know to do this, nor should it. Essentially if we're depending on any native deps, I need to know what the full set of links flags are for those native deps. Maybe a Bazel dev could point me in the right direction to find those link flags.